lib/article_json/export/html/elements/text.rb

```diff
@@ -36,8 +36,17 @@ def bold_and_italic_node
 
           # @return [Nokogiri::XML::NodeSet]
           def content_node
-            return create_text_node(@element.content) if @element.href.nil?
-            create_element(:a, @element.content, href: @element.href)
+            return create_text_nodes(@element.content) if @element.href.nil?
+            create_element(:a, href: @element.href) do |a|
+              a.add_child(create_text_nodes(@element.content))
+            end
+          end
+
+          # @param [Nokogiri::HTML::NodeSet] text
+          def create_text_nodes(text)
+            Nokogiri::HTML
+              .fragment(text.gsub(/\n/, '<br>'))
```

Comment:
we don't want to add this to the super class's method, do we?

Reply:
I think it's good to only have it in here since only text nodes support line breaks at the moment